pkgs/by-name/fi/filebot/package.nix

```
@@ -63,9 +66,22 @@ stdenv.mkDerivation (finalAttrs: {
       --replace '$FILEBOT_HOME/data/.license' '$APP_DATA/.license' \
       --replace '-jar "$FILEBOT_HOME/jar/filebot.jar"' '-Dcom.googlecode.lanterna.terminal.UnixTerminal.sttyCommand=${coreutils}/bin/stty -jar "$FILEBOT_HOME/jar/filebot.jar"'
     wrapProgram $out/opt/filebot.sh \
-      --prefix PATH : ${lib.makeBinPath [ openjdk17 ]}
+      --prefix PATH : ${
+        lib.makeBinPath [
+          chromaprint
+          openjdk17
+          p7zip
+          unrar
+        ]
+      }
     # Expose the binary in bin to make runnable.
     ln -s $out/opt/filebot.sh $out/bin/filebot
+
+    # Delete the built-in libmediainfo so the program has to use the symlinked nixpkgs version
+    rm $out/opt/lib/**/libmediainfo.so
+    ln -s ${libmediainfo.outPath}/lib/libmediainfo.so $out/opt/lib/Linux-aarch64/libmediainfo.so
+    ln -s ${libmediainfo.outPath}/lib/libmediainfo.so $out/opt/lib/Linux-armv7l/libmediainfo.so
+    ln -s ${libmediainfo.outPath}/lib/libmediainfo.so $out/opt/lib/Linux-x86_64/libmediainfo.so
   '';
```

Perhaps we should add a few tests for some functionality.
Only problem is that renaming commands require a license, but I found that `-mediainfo` with `--filter` works to test  the `libmediainfo` integration:

```suggestion
  '';
  
  doInstallCheck = true;
  installCheckPhase = ''
    runHook preInstallCheck

    mkdir ./in
    ${ffmpeg}/bin/ffmpeg -f lavfi -i color=color=red -t 0.1 -vf scale=2:2 ./in/"Doctor Who (2005) s01e01.mkv"
    ${ffmpeg}/bin/ffmpeg -f lavfi -i color=color=red -t 0.1 -vf scale=2:2 ./in/"Doctor Who (2005) s01e02.mkv"
    ${ffmpeg}/bin/ffmpeg -f lavfi -i color=color=red -t 0.1 -vf scale=2:2 ./in/"Doctor Who (2005) s01e03.mkv"

    export HOME=$(mktemp -d)
    $out/bin/filebot -mediainfo -r ./in --filter "f.video && minutes < 15"

    runHook postInstallCheck
  '';
```

---

Also, we should add `preInstall` & `postInstall` hooks to `installPhase` as well.